### ISSUE TITLE
Ukázat v lekci vnořené podmínky

### ISSUE THREAD

**Issue by ZelenyMartin** [podlomar/uvod-do-progr#10]
* Obohatit lekci https://kodim.cz/czechitas/uvod-do-progr/prvni-krucky/podminky o ukázku vnořených podmínek (např. přidat kontrolu zůstatku na účtu)
* Vnořené podmínky: přidat výpis "dospělý"/"dítě"
```
if vek < 18:
  print('Dítě')

  if vek < 6:
    print('Předškolák')
  elif vek < 15:
    print('Školák')
  else:
    print('Mladistvý')
else:
  print('Dospělý')
```

* Přidat kontrolu na rovnost, např.
```
if vek == 0:
  print('Batole')
```

**Comment by pesikj** [comment]
@ZelenyMartin 
Navrhuji jiný příklad

```
vek = input("Zadej věk: ")
if vek.isdigit():
  vek = int(vek)
  if vek < 6:
    print('Předškolák')
  elif vek < 18:
    print('Nezletilý')
  else:
    print('Dospělý')
else:
  print('Věk musí být číslo')
```

- Vyřešíš tím dotaz, který padá skoro vždycky: Co když tam uživatel nezadá číslo?
- Rozmezí a hodnoty ve vnořené podmínce je možné libovolně upravit.

**Comment by ZelenyMartin** [comment]
Jo, je to mnohem lepsi priklad. Ale jeste tam pridam pretypovani promenne `vek` na `int`.

**Comment by pesikj** [comment]
Jo, to jsem zapomněl :-)

**Comment by ZelenyMartin** [comment]
@pesikj Ahoj, me uz se tam ted nejak moc nechce ukazovat to `vek.isdigit()` - metody objektu, funkce vracejici `bool`. Radeji bych ten text zjednodusoval nez naopak. Taky tam neni nikde moc povidani o `str()` a `int()` a ve cvicenich na to trochu narazi, kdyz to zapomenu explicitne zminit. Radeji bych tam dodal to.

**Comment by pesikj** [comment]
@ZelenyMartin Ale potřebujeme jim ukázat vnořenou podmínku, protože ta je potřeba na řešení cvičení.
